README.md

```diff
@@ -219,6 +224,42 @@ All available methods and attributes of `WMTOperations` API are:
 
 For more details on the API, visit [`WMTOperations` code documentation](https://github.com/wultra/mtoken-sdk-ios/blob/develop/WultraMobileTokenSDK/Operations/WMTOperations.swift).
 
+#### WMTUserOperation
+
+Operations objects retrieved through the online API (like `getOperations` method in `WMTOperations`) are called "user operations".
+
+Under this abstract name, you can imagine for example "Login operation", which is a request for signing in to the online account in a web browser on another device. **In general, it can be any operation that can be either approved or rejected by the user.**
+
+Visually, the operation should be displayed as an info page with all the attributes (rows) of such operation, where the user can decide if he wants to approve or reject it.
+
+Properties of the `WMTUserOperations`:
+
+```swift
+class WMTUserOperation {
+
+	/// Unique operation identifier
+	public let id: String
+	    
+	/// System name of the operation (for example login).
+	public let name: String
```

Comment:
I think we could also add some guidelines, such as:

- `name` - This property lets you adjust the UI for various operation types. For example, the "login" operation may display a specialized interface with an icon or an illustration, instead of an empty list of attributes, "payment" operation can include a special icon that denotes payments, etc.

Reply:
Yop